lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -1002,11 +1021,83 @@ static bool IsSwiftReferenceType(ValueObject &object) {
   return false;
 }
 
+static bool printObjectViaPointer(Stream &strm, ValueObject &object,
```

Comment:
Should this function return llvm::Error and return createStringError for the various failure modes for easier debugging?

Reply:
I'm not able to picture how this improves debugging? Can you walk me through it?